src/pybind/mgr/dashboard/controllers/health.py

```diff
@@ -106,15 +106,13 @@ def fs_map(self):
         fs_map = mgr.get('fs_map')
         if self._minimal:
             fs_map = self._partial_dict(fs_map, ['filesystems', 'standbys'])
-            fs_map['standbys'] = [{}] * len(fs_map['standbys'])
             fs_map['filesystems'] = [self._partial_dict(item, ['mdsmap']) for
                                      item in fs_map['filesystems']]
             for fs in fs_map['filesystems']:
                 mdsmap_info = fs['mdsmap']['info']
                 min_mdsmap_info = dict()
                 for k, v in mdsmap_info.items():
                     min_mdsmap_info[k] = self._partial_dict(v, ['state'])
-                fs['mdsmap'] = dict(info=min_mdsmap_info)
         return fs_map
 
```

Comment:
Shouldn't line 129 (previous 131) be removed too?

(Just read in the comments that you plan to remove it.)

Reply:
Yes , I guess! Thanks for pointing out! Will make the changes